src/main/fc/rc_modes.c

```diff
@@ -41,7 +42,8 @@
 
 boxBitmask_t rcModeActivationMask; // one bit per mode defined in boxId_e
 
-static bool paralyzeModeEverDisabled = false;
+static boxBitmask_t STICKY_MODES = {{0, 256}}; // Paralyze
```

Comment:
Not sure about assigning the bitmasks in this way - we should really calculate it from BOXPARALYZE.

Reply:
@mikeller fixed